TaxonomyService.md

```diff
@@ -0,0 +1,61 @@
+Taxonomy Service
+=============
+
+Table of Contents
+_________
+- [Request Information](#request-information)
+- [Sample Response](#sample-response)
+- [Available Taxonomies](#available-taxonomies)
+- [Versioning](#versioning)
+
+
+
+#Request Information
+
+
+HTTP method: GET
+Parameters (query/form):
+-        No Parameters
+ 
+Example: https://api.careerbuilder.com/core/taxonomy/ ([Desired Taxonomy](#available-taxonomies))/
+
+#Sample Response
+
+
+```
+{
+    "data": {
+        "en": [
+            {
+                "id":"0",
+                "title":"Title0",
+                "description":"Description0",
+                "type":"type"
+            },
+            {
+                "id":"1",
+                "title":"Title1",
+                "description":"Description1",
+                "type":"type"
+            }
+        ]
+    }
+}
+```
+
+
+#Response Information
+----------
+The response returns a single data node which contains a list of languages. Each language contains a list of taxonomies within that language.
+
```

Comment:
also, I think you should move the response description before the example response.